index.html

```diff
@@ -10,14 +10,14 @@
 <body>
     <header>  
           <h1>gerrit</h1>
-           <nav class="nav">
-            <a href="#myhobbies" class="navelements">hobbies</a>  
-            <a href="#contact" class="navelements">contact</a>
-            <a href="#biography"class="navelements">biography</a>
-            <a href="#cv" class="navelements">cv</a>
+           <nav class="navigation">
+            <a href="#myhobbies" class="navigation__navigation--item">hobbies</a>  
+            <a href="#contact" class="navigation__navigation--item">contact</a>
+            <a href="#biography"class="navigation__navigation--item">biography</a>
+            <a href="#cv" class="navigation__navigation--item">cv</a>
         </nav> 
 
-        <figure class="justin-class center">
+        <figure class="figure center">
             <img src="https://www.pngall.com/wp-content/uploads/2016/06/Justin-Bieber-Download-PNG.png"
                  alt="" width="250" length="200">
             <figcaption style="padding-left: 30px;">Justin Bieber, not me</figcaption>
```

Comment:
```suggestion
            <figcaption >Justin Bieber, not me</figcaption>
```

here you are using inline CSS, try to always avoid this.